src/sentry/search/events/builder/discover.py

```diff
@@ -1555,6 +1556,29 @@ def resolve_query(
         pass
 
 
+class ReleaseHealthQueryBuilder(UnresolvedQuery):
```

Comment:
Please put this either in builder/release_health.py or just put it directly in src/sentry/snuba/metrics/query_builder.py